photon-server/src/main/resources/web/index.html

```diff
@@ -1 +1,14 @@
-<p>UI has not been copied!</p>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="./favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Photon Client</title>
+    <script type="module" crossorigin src="./assets/index-CIrfZRaJ.js"></script>
+    <link rel="stylesheet" crossorigin href="./assets/index-Bt-HXv2B.css">
+  </head>
+  <body>
+    <div id="app"></div>
+  </body>
+</html>
```

Comment:
Accepting this suggestion will revert it to the version in master:
```suggestion
<p>UI has not been copied!</p>
```